e2e-tests/mdx/src/pages/regressions/index.mdx

```diff
@@ -0,0 +1,7 @@
+This is a regression test for query extraction bug in Gatsby.
+
+All of the following is required to hit this bug:
+
+1. There should be at least one other plugin implementing `preprocessSource`
+2. This plugin must be listed before `gatsby-plugin-mdx` in `gatsby-config.js`
+3. MDX file must contain word `graphql`
```

Comment:
I assume `graphql` word in this file is important to this test - should maybe add comment saying to not mess with content of this file (or at very least not remove `graphql` word)? Just to add context for any future person that would stumble on this file